client/src/amplify-ui/block-switcher/block-switcher.tsx

```diff
@@ -49,24 +59,43 @@ export class AmplifyBlockSwitcher {
     }
   }
 
+  handleInitSelectedTabHeading() {
+    if (this.tabHeadings?.length > 0 && this.selectedTabHeadings?.length > 0) {
+      for (const heading of this.selectedTabHeadings) {
+        if (this.tabHeadings.includes(heading)) {
+          this.activeChildI = this.tabHeadings.indexOf(heading);
+          return;
+        }
+      }
+    } else {
+      if (typeof window === "undefined") {
+        this.activeChildI = 0;
+      } else {
+        this.activeChildI = 0;
```

Comment:
Collapse to just `this.activeChildI = 0` with no `if` statement?  Or is this second one supposed to be something different...

Reply:
Yeah. This should be cleaned up. Originally, I was setting the value to some nonsense number if outside of browser context, but then switched back and forgot to clean up.